src/config.ts

```diff
@@ -288,6 +288,16 @@ export const enum GitCommandSorting {
 	Usage = 'usage',
 }
 
+export const enum GraphScrollMarkerTypes {
```

Comment:
See `package.json` comments

Reply:
Done in commit: [Pluralizes some marker names](https://github.com/gitkraken/vscode-gitlens/pull/2489/commits/a52b4818373288955a200db90b50b72ba210eb97)